dags/app/twitter_post_notification_bot/udf.py

```diff
@@ -0,0 +1,62 @@
+import requests
+from airflow import settings
+from sqlalchemy.orm import sessionmaker
+
+
+def main() -> None:
+    url = "https://twitter135.p.rapidapi.com/v2/UserTweets/"
+    # 499339900 is PyConTW's twitter id
+    querystring = {"id": "499339900", "count": "1"}
+    headers = {
+        "X-RapidAPI-Key": Variable.get("RAPIDAPIAPI_KEY"),
+        "X-RapidAPI-Host": "twitter135.p.rapidapi.com",
+    }
+    webhook_url = Variable.get("DISCORD_POST_NOTIFICATION_WEBHOOK")
+    response = requests.get(url, headers=headers, params=querystring)
+    response_json = response.json()
+    try:
+        Session = sessionmaker(bind=settings.engine)
+        # Update the variable using a context manager
+        variable_key = "TWITTER_LATEST_REST_ID"
```

Comment:
use this `TWITTER_LATEST_REST_ID` as a cursor, to point to the latest tweet that we've sent to discord